addons/comments/stringtable.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project name="ZEN">
+    <Package name="Comments">
+        <Key ID="STR_ZEN_Comments_DisplayName">
+            <English>Comments</English>
+            <Czech>Poznámky</Czech>
+            <German>Kommentare</German>
+            <Russian>Комментарии</Russian>
+            <Polish>Komentarze</Polish>
+            <Italian>Commenti</Italian>
+            <Spanish>Comentarios</Spanish>
+            <French>Commentaires</French>
+            <Chinese>評論</Chinese>
+            <Korean>댓글</Korean>
+            <Japanese>コメント</Japanese>
+            <Portuguese>Comentários</Portuguese>
+            <Chinesesimp>评论</Chinesesimp>
+            <Turkish>Yorumlar</Turkish>
+        </Key>
```

Comment:
This is the same as `STR_3DEN_Comment_textPlural`. Should I remove this entry and use `STR_3DEN_Comment_textPlural` for the display name?

Reply:
Yes.